lib/libc/string/aarch64/bcopy.S

```diff
@@ -278,7 +278,7 @@ backward_tiny:
 	ret
 9:
 	/* length is small(<32), and src or dst may be unaligned */
-	eor	TMP_X, SRC0, DST0
+	eor	TMP_X, SRC0, DST
```

Comment:
Since this attempts to fix a bug that is possibly present in NetBSD please provide short explanation before this line, i.e. what are the symptoms, when the bug is triggered and why this change is supposed to fix it.